src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -32,13 +24,10 @@ private void ScanStringLiteral(ref TokenInfo info, bool inDirective)
                 // Normal string & char constants can have escapes. Strings in directives cannot.
                 if (ch == '\\' && !inDirective)
                 {
-                    char c2;
-                    ch = this.ScanEscapeSequence(out c2);
+                    ch = this.ScanEscapeSequence(out var c2);
                     _builder.Append(ch);
                     if (c2 != SlidingTextWindow.InvalidCharacter)
-                    {
```

Comment:
While I understand that there are some exceptions in the compiler codebase, our guideline is still to keep braces on `if` statements. This file seems to have been consistent in using them, so we should stick with that. Also, it's not necessary to this PR.
Same applies to other instances below. Thanks

nit: I'd keep the explicit type as well, since `char` is barely longer than `var` anyways.

Reply:
> While I understand that there are some exceptions in the compiler codebase, our guideline is still to keep braces on if statements

Sure.  I can add these back in.  However, my general feedback is that hte compiler codebase tends to be pretty 'all over the map'.  It would be great if some consistency passes could be done to do it so that code would be definitively in-line or out-of-line with the patterns of the rest.  :)